lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -752,13 +752,17 @@ void InterpreterFunction::fwdGatherInst(const glow::GatherInst *I) {
   // Calculate the size of each sample in the batch.
   size_t numSamples = (dataT->size() * elementSize) / dataSampleSize;
 
+  // Calculate number of samples in the batch.
+  size_t batchSize = dataTy.dims()[batchDims];
```

Comment:
I believe we would need a `(void)batchSize` to avoid release build to complain about an unused variable.